zerver/openapi/zulip.yaml

```diff
@@ -1692,6 +1692,8 @@ paths:
                                     has to be muted, the second element is the topic name to be muted
                                     and the third element is an integer UNIX timestamp representing
                                     when the topic was muted.
+                                    **Changes**: Made a 3-item tuple in Zulip 3.0(feature level 1). Previously
```

Comment:
We spell these references as "Zulip 3.0 (feature level 1)" -- note the space.

Also "Previously, they were a ...", note the comma.